demo-forms/custom_contacts/README.MD

```diff
@@ -0,0 +1,11 @@
+# Applying Sample Custom Contact Types
+
+
+1. Download the current `app_settings.json`
+1. Replace the existing `contact_types` with the contents of [contact_types.json](contact_types.json)
+1. Upload the change to `app_settings.json`
+1. Download the english language file.
+1. Add the translation keys in [en_translations](en_translations.properties)
```

Comment:
Ah, yes, looks like it was [that file ](https://github.com/medic/cht-core/blob/master/config/default/translations/messages-en.properties);)

```suggestion
1. Edit the `messages-en.properties` file and append the translation keys in [en_translations](en_translations.properties) at the bottom of the  `messages-en.properties`  file.
```